Assets/SEE/Dissonance/ChatInputController.cs

```diff
@@ -44,16 +52,28 @@ public class ChatInputController
         /// Registers <see cref="OnInputEndEdit(string)"/> to be called when the user
         /// has ended his/her input.
         /// </summary>
-        private void Start ()
+        protected override void Start ()
         {
-            Comms = Comms ?? FindObjectOfType<DissonanceComms>();
+            base.Start();
 
-            inputField = GetComponentsInChildren<InputField>().Single(a => a.name == "ChatInput");
+            // Find the input field.
+            // Note: GetComponentsInChildren recurses into all transitive descendants.
+            inputField = GetComponentsInChildren<InputField>(true).Single(a => a.name == chatInputName);
+            if (inputField == null)
```

Comment:
This can never happen, since `Single` is used. Either `SingleOrDefault` should be used, or this if condition should be removed.

Reply:
Fixed.